src/taco-cli/cli/simulate.ts

```diff
@@ -0,0 +1,50 @@
+/**
+ ? *******************************************************
```

Comment:
Looks like you might have been bitten by the BOM; at least I assume that's why there's a bunch of question marks here.